PrBoom-Plus_2_5_1_6.sh

```diff
@@ -16,6 +16,21 @@ DOOM_EXE_NAME="prboom-plus"
 DOOM_OPTIONS=""
```

Comment:
For my modifications to work, "-iwad " needs to be added to the DOOM_OPTIONS="" line. doom.wad (from Doom 1) otherwise tries to load as a pwad and loads incorrectly.

That said, this script only works for IWADs... maybe it's worthwhile to have a separate directory for IWADs and PWADs so that players can load whichever game they prefer with whatever options they prefer?